Overhaul of plot_bucell_nuclide_network utility

Rademacher opened the pull request:
Made some improvements to the plot_bucell_nuclide_network utility function:
Threshold can now be given as fraction of the total reaction rate instead of absolute reaction rate (the latter is still the default). The nodes in the created plot now form a proper circle, and the plot is created quadratic to avoid distortion of dimensions. Readibility of text has been improved. The plot is now saved for further use. Deleted some useless code snippets.

Rademacher committed:
Overhaul of plot_bucell_nuclide_network utility

Made some improvements to the plot_bucell_nuclide_network utility function:
Threshold can now be given as fraction of the total reaction rate instead of absolute reaction rate (the latter is still the default). The nodes in the created plot now form a proper circle, and the plot is created quadratic to avoid distortion of dimensions. Readibility of text has been improved. The plot is now saved for further use. Deleted some useless code snippets.

jung-benjamin commented on onix/utils/data_processor.py:
```diff
@@ -143,27 +156,23 @@ def plot_bucell_nuclide_network(nuclide, step, path, cell, threshold):
             #   continue    
-            if reaction_val < threshold:
-                continue
+            if threshold_mode=='absolute':
```
```suggestion
            if threshold_mode == 'absolute':
```

Rademacher committed:
Update onix/utils/data_processor.py

Co-authored-by: jung-benjamin <63796898+jung-benjamin@users.noreply.github.com>

Rademacher committed:
Update onix/utils/data_processor.py

Co-authored-by: jung-benjamin <63796898+jung-benjamin@users.noreply.github.com>

Rademacher committed:
Update onix/utils/data_processor.py

Co-authored-by: jung-benjamin <63796898+jung-benjamin@users.noreply.github.com>

Rademacher committed:
Update onix/utils/data_processor.py

Co-authored-by: jung-benjamin <63796898+jung-benjamin@users.noreply.github.com>

Rademacher committed:
Update onix/utils/data_processor.py

Co-authored-by: jung-benjamin <63796898+jung-benjamin@users.noreply.github.com>

Rademacher committed:
Update onix/utils/data_processor.py

Co-authored-by: jung-benjamin <63796898+jung-benjamin@users.noreply.github.com>

Rademacher committed:
Update onix/utils/data_processor.py

Co-authored-by: jung-benjamin <63796898+jung-benjamin@users.noreply.github.com>

Rademacher committed:
Added option to disable show

Added option to disable plt.show() to facilitate plot creation in the background

jung-benjamin commented:
We should not merge this, because it has already been merged into the master branch.

jung-benjamin commented:
Sorry, the above comment was incorrect. The branch hast not been merged yet. However, a diverging history will make merging complicated, so we should resolve this before merging.
